frontend/packages/webterminal-plugin/integration-tests/support/step-definitions/pages/web-terminal/webTerminal-page.ts

```diff
@@ -46,7 +46,7 @@ export const webTerminalPage = {
 
   deleteTerminalInstanceActionMenu: () => {
     cy.byLegacyTestID('actions-menu-button').should('be.visible').click();
-    cy.byLegacyTestID('action-items').should('be.visible');
+    cy.get('[role="menu"]').should('be.visible');
```

Comment:
can we use data-test ids?  as i see e2e failures with this change in 4.19 and good to rely less on PF 

cc @lokanandaprabhu